### ISSUE TITLE
Gather more metrics about the Grafana project

### ISSUE THREAD

**Issue by bergquist** [grafana/grafana#11163]
The insights we get from Github is quite limited. We often asked ourself about or issue/PR rate and how we can improve it. It would be especially interesting to see how these numbers are effect by certain events outside the github repo. 

Before we start working on an implementation/looking for other solutions, let's define the questions we hope this tool can help us with. 

- [ ] 15th/50th/95th time for new PR/Issue until merge/closed
- [ ] Active issues. Total and per label. So we can track the popularity of certain labels. And also identify unused labels.  
- [ ] days since last response/decision from maintainer on PR's
- [ ] Show PR/issues that passed our feedback SLO.

Any other questions we should consider before evaluating different solutions?

ping @grafana/grafana-labs-core-devs

**Comment by marefr** [comment]
Just came to think about that we probably would like possibility to look at contributions by only external contributors/non-grafana labs team, by only core grafana labs team or both.

**Comment by alexanderzobnin** [comment]
Maybe it makes sense to track also individual features like we track datasources stats. But it should be gently to avoid pulling any sensitive data.
